torch_xla/csrc/runtime/pjrt_computation_client.cc

```diff
@@ -677,8 +677,9 @@ PjRtComputationClient::ExecuteReplicated(
   TF_VLOG(5) << "ExecuteReplicated acquiring PJRT device lock for "
              << spmd_device_str << " Done";
 
-  std::optional<std::vector<xla::PjRtFuture<xla::Status>>> returned_futures(
-      devices.size());
+  std::optional<std::vector<xla::PjRtFuture<xla::Status>>> returned_futures =
+      std::vector<xla::PjRtFuture<xla::Status>>();
+  returned_futures->reserve(devices.size());
```

Comment:
Can you also remove the `reserve` here? Let's let PJRT handle this vector.

Reply:
ok, I've removed it